### DISCUSSION TITLE
Unable to view certain elastic fleet integrations

### DISCUSSION THREAD

**Discussion by mckenziemmack** [Security-Onion-Solutions/securityonion#12985] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

251G

### Storage for /

558G

### Storage for /nsm

38T

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am having an issue with the Elastic Agent integrations. This is an error that I receive when clicking on certain integrations.

![Screenshot 2024-05-09 140329](https://github.com/Security-Onion-Solutions/securityonion/assets/116289712/c0904b30-7ac7-4ba6-b2f6-cf39c9121ef9)

Under `/opt/so/conf/elastic-fleet/integrations` I am only seeing 4 options, when in Security Onion Console - Fleet I am seeing the full 10+ integrations that I should have. I am able to view and edit those 4 integrations on SOC without error, but the ones that I am not seeing in SOC are giving me the error message above.

Any idea what the problem could be or where I should look into? 

Thank you!

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
Are there any errors when you run 
```sudo so-restart elasticfleet```

**Reply by mckenziemmack** [reply]
Yes, however, to restart Elastalert I have been using `sudo so-elastalert-restart` without issues.

![Screenshot 2024-05-10 095801](https://github.com/Security-Onion-Solutions/securityonion/assets/116289712/aaec3508-20eb-48ce-8981-2424b843e2df)

![image_2024-05-10_095907761](https://github.com/Security-Onion-Solutions/securityonion/assets/116289712/0aec29b9-8953-4c17-882c-460b12828eed)
